charts/internal/shoot-control-plane/templates/network-policies.yaml

```diff
@@ -51,6 +51,23 @@ spec:
 ---
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
+metadata:
```

Comment:
When you add this to the shoot control plane as well, please also add it to the chart objects of the control plane shoot chart: https://github.com/metal-stack/gardener-extension-provider-metal/blob/master/pkg/controller/controlplane/valuesprovider.go#L224

Reply:
done